app/lib/pages/map/page.tsx

```diff
@@ -119,7 +119,7 @@ const Page = () => {
 
           scrollToElement: true,
           dontShowAgain: true,
-          skipLabel: "Skip",
+          skipLabel: "Skip", //change the look of this button
```

Comment:
yes; the skip button needs to be moved to the left

this is probably accomplished in the intro.js popup